### PR TITLE
chore: convert and update 'developer/17.0/guides/develop/touch/...'

### PR DESCRIPTION
**Pull request by Meng-Heng**
Making changes to this folder as requested by Marc (https://community.software.sil.org/t/another-touch-screen-developer-oddity/8358/4).

### CODE REVIEW

File: developer/17.0/guides/develop/touch-keyboard-tutorial/making-touch-keyboard.md [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+Now there is still one very important thing I need to do. If you have
+ever designed desktop keyboards in the layout view, you would think
+putting the character you want on the keycap is all you need to do. But
+mobile keys don’t work that way. (Technically there is a difference
+between a physical keyboard, where pressing a key sends a unique code to
+the computer, and a virtual keyboard, where tapping a key only registers
+a tap on an arbitrary part of the screen). In adding a mobile key, we
+have to enter what the key does, not just add a character to the keycap.
+To make our new key insert the Unicode character 01B4, I’ll go to the
+“code” box and change the string to “U\__01B4”. (“U\__” followed by a
```
Comment (darcywong00) [inline developer/17.0/guides/develop/touch-keyboard-tutorial/making-touch-keyboard.md:159]: I don't think we need to escape underscores in Markdown
```suggestion
“code” box and change the string to “U_01B4”. (“U_” followed by a
```

File: developer/17.0/guides/develop/touch-keyboard-tutorial/making-touch-keyboard.md [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+&gt; Compile Keyboard, or press F7). Hopefully you’ll see it compiled
+with no error messages:
+
+![successful
+compile](/cdn/dev/img/developer/100/simpleTouchKeyboard_20.png)
+
+If you see a warning like:
+
+```
+***Warning: line 0  warning 2092: Key " T_new_579" on layer "default", platform "phone", is a custom key but has no corresponding rule in the source.***
```
Comment (darcywong00) [inline developer/17.0/guides/develop/touch-keyboard-tutorial/making-touch-keyboard.md:269]: I'm not seeing any styling applied with the `***` in the code blocks. Maybe we just remove them?
```suggestion
Warning: line 0  warning 2092: Key " T_new_579" on layer "default", platform "phone", is a custom key but has no corresponding rule in the source.
```

File: developer/17.0/guides/develop/touch-keyboard-tutorial/making-touch-keyboard.md [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+```
+
+This shows you forgot to enter the correct keycode for a new key you
+added. Double check your longpress popup keys for one with a “T_new_NNN”
+code, and add in the desired “U_NNNN” code.
+
+You might also see a message like
+
+```
+***Error: line 0  error 405A: Key " U+0181" on "phone", layer "default" has an invalid identifier.***
```
Comment (darcywong00) [inline developer/17.0/guides/develop/touch-keyboard-tutorial/making-touch-keyboard.md:279]: same here
```suggestion
Error: line 0  error 405A: Key " U+0181" on "phone", layer "default" has an invalid identifier.
```

**Review by Meng-Heng — COMMENTED** [review]
Hello @darcywong00, I have applied the changes. Thank you for taking a look at these files.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**